vivisect/impemu/emulator.py

```diff
@@ -20,14 +20,26 @@
 init_stack_size = 0x8000
 init_stack_map = b'\xfe' * init_stack_size
 
-def imphook(impname):
 
+def _get_args(kwargs_value, config_value, default=None):
```

Comment:
i'm not a fan of this name.  can you think of a different function name which describes the gist of what it does better than `_get_args`?

Reply:
I will pull this change out into a PR to be created against the master branch as we discussed.

Reply:
fix the docstring and we'll call it good.  make it give more details about the use-case, how it's helpful, how to use it.

Reply:
how does this look
```
def _get_args(kwargs_value, config_value, default=None):
    '''
    Returns a configuration value to be used by the workspace emulator. The
    priority is:
    1. Any kwargs provided to the WorkspaceEmulator class initializer.
    2. A value from the vivisect configuration.
    3. The default value.
    '''
    if kwargs_value is not None:
        return kwargs_value
    elif config_value is not None:
        return config_value
    else:
        return default
```

Reply:
renamed to `_priority_get_args`